spec/system/user_sign_up_spec.rb

```diff
@@ -75,4 +76,16 @@ def expect_to_be_on_sign_up_page
     expect_to_be_on_sign_up_page
     expect(page).to have_css('.alert')
   end
-end
\ No newline at end of file
+
+  scenario 'and disables notifications' do
```

Comment:
I'd just stick to the conventions of all the scenarios in this spec, so I'd change this to `'with notifications disabled'`.

Sorry for the nit-pick 😅🤓

Reply:
You're super right. I was naming the spec according to another scenario in my mind (one which the user goes to disable his notifications), nice catch !